### ISSUE TITLE
fish_update_completions broken?

### ISSUE THREAD

**Issue by Vintodrimmer** [voidlinux/void-packages#6036]
Good day,

I've had some problems with very slow completions and decided to run `fish_update_completions` and got

```
Traceback (most recent call last):
  File "/usr/share/fish/tools/create_manpage_completions.py", line 929, in <module>
    file_paths.extend(get_paths_from_manpath())
  File "/usr/share/fish/tools/create_manpage_completions.py", line 860, in get_paths_from_manpath
    proc = subprocess.Popen(['manpath'], stdout=subprocess.PIPE)
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

I have python 2.7 installed:

```
xbps-query -s python
[*] python-2.7.13_3         Interpreted, interactive, object-oriented programming language
[*] python-alsa-1.0.29_2    Official Python bindings for ALSA
[*] python-cairo-1.10.0_5   Python2 bindings for the cairo graphics library
[*] python-gobject-3.22.0_1 Python2 bindings for GObject
[*] python3-3.5.3_2         Interpreted, interactive, object-oriented programming language (3.5 series)
[*] youtube-dl-2017.03.24_1 CLI program to download videos from YouTube and other sites (Python2)
```

Is there some  problem with the fish-shell or am I just missing something?

**Comment by Duncaen** [comment]
Looks like it only works with `man-db`, void uses `mdocml` by default.
https://github.com/fish-shell/fish-shell/blob/11a60c8374c21c299bab35ae12a30b54a44bc9e3/share/tools/create_manpage_completions.py#L860
It uses `manpath` from `man-db`.

**Comment by Vintodrimmer** [comment]
Thank you.

Is it reasonable to ask here for a void-specific patch?
Or is it better to ask fish-guys to check for mdocml?

**Comment by faho** [comment]
This has been fixed in fish's master branch since 03e0f1eb21e9df76ed792cbd8349d04ec29b8906. You'll want to backport 03e0f1eb21e9df76ed792cbd8349d04ec29b8906, 94b71353ef16e90db850431fa226ff7c33939254 and b3d5fb71f484de4dae8103ab4ac099377b3b698c (i.e. the current state of create_manpath_completions.py, which shouldn't depend on any other changes in what will become 2.6.0) or wait for the next release.
